src/stack.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+export default class Stack {
+
+  constructor(data) {
+    this.top = null;
+    this.size = 0;
+  }
+
+  push(element) {
+    let nextNode = new Node (element)
+    nextNode.next = this.top
+    this.top = nextNode
+    this.size++
+  }
+
+  pop() {
+    if(this.top) {
+      let popped = this.top;
+      this.top = this.next;
+      this.size -= 1;
+      return popped.data;
+    } else {
+      return null;
+    }
+  }
+
+  peek() {
+    if(this.top) {
```

Comment:
You can simply just return this.top. Should be null if queue is empty, the if statement is redundant.